certified-connectors/OrbusInfinity/apiProperties.json

```diff
@@ -0,0 +1,70 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
```

Comment:
Please modify from oAuthSetting to oauthSetting.  Kindly refer the schema here. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

Reply:
fixed